### ISSUE TITLE
Clarify dependencies

### ISSUE THREAD

**Issue by dmethvin** [iheanyi/facebook-page-unliker#3]
Several things appear to be required that aren't installed by `npm install` or mentioned in the readme. For example, you must `npm install -g ember-cli`. That leads to more messages about missing dependencies when launching ember-cli.

**Comment by dmethvin** [comment]
Just to be sure I got everything right I tried a reinstall after installing ember-cli. Bower gives me this conflict:
```
Unable to find a suitable version for handlebars, please choose one:
    1) handlebars#~1.3.0 which resolved to 1.3.0 and is required by page-unliker
    2) handlebars#>= 1.0.0 < 2.0.0 which resolved to 1.3.0 and is required by ember#1.8.1
    3) handlebars#>= 2.0.0 < 3.0.0 which resolved to 2.0.0 and is required by ember#1.9.1, ember#1.9.1, ember#1.9.1, ember#1.9.1, ember#1.9.1
```
I chose 1 (handlebars 1.3.0) but that leads to the app failing to run:
```
"DEBUG: -------------------------------" vendor.js:28585
"DEBUG: Ember                   : 1.8.1" vendor.js:28585
"DEBUG: Ember Data              : 1.0.0-beta.12" vendor.js:28585
"DEBUG: Handlebars              : 1.3.0" vendor.js:28585
"DEBUG: jQuery                  : 1.11.2" vendor.js:28585
"DEBUG: Ember Simple Auth       : 0.7.2" vendor.js:28585
"DEBUG: Ember Simple Auth Torii : 0.7.2" vendor.js:28585
"DEBUG: -------------------------------" vendor.js:28585
"No authorizer was configured for Ember Simple Auth - specify one if backend requests need to be authorized." vendor.js:28585
"generated -> route:index.index" Object { fullName: "route:index.index" } vendor.js:28585
"generated -> controller:application" Object { fullName: "controller:application" } vendor.js:28585
"Rendering application with default view <page-unliker@view:toplevel::ember398>" Object { fullName: "view:application" } vendor.js:28585
"Rendering index with default view <page-unliker@view:default::ember406>" Object { fullName: "view:index" } vendor.js:28585
"generated -> controller:index.index" Object { fullName: "controller:index.index" } vendor.js:28585
"Could not find "index.index" template or view. Nothing will be rendered" Object { fullName: "template:index.index" } vendor.js:28585
```

**Comment by iheanyi** [comment]
Definitely should be Handlebars 2.0, I'll fix that. Thanks for pointing it ou!
